kanidmd/src/lib/lib.rs

```diff
@@ -1,6 +1,7 @@
 //! The Kanidmd server library. This implements all of the internal components of the server
 //! which is used to process authentication, store identities and enforce access controls.
 
+#![recursion_limit = "512"]
```

Comment:
Is this related to Oauth2?

Reply:
it's related to compliing of the code, I think we are getting a lot of macro recursion, but it may have been a bug too in something. Let me rlemove it and retest. 

```

error: recursion limit reached while expanding `__lazy_static_internal!`
   --> kanidmd/src/lib/constants/uuids.rs:201:1
    |
201 | / lazy_static! {
202 | |     pub static ref UUID_ADMIN: Uuid = Uuid::parse_str(STR_UUID_ADMIN).unwrap();
203 | |     pub static ref UUID_IDM_ADMIN: Uuid = Uuid::parse_str(STR_UUID_IDM_ADMIN_V1).unwrap();
204 | |     pub static ref UUID_DOES_NOT_EXIST: Uuid = Uuid::parse_str(STR_UUID_DOES_NOT_EXIST).unwrap();
...   |
322 | |         Uuid::parse_str(STR_UUID_SCHEMA_CLASS_OAUTH2_RS_BASIC).unwrap();
323 | | }
    | |_^
    |
    = help: consider adding a `#![recursion_limit="256"]` attribute to your crate (`kanidm`)
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

There it is, it's because of the use of lazy_static in constants.